website/src/home/content/privacy-policy.md

```diff
@@ -0,0 +1,33 @@
+# Privacy Policy
+
+Compa takes your privacy seriously. To better protect your privacy we provide this privacy policy notice explaining the way your personal information is collected and used.
+
+## Collection of Routine Information
+
+This website track basic information about their users. This information includes, but is not limited to, IP addresses, browser details, timestamps and referring pages. None of this information can personally identify a specific user to this website. The information is tracked for routine administration and maintenance purposes.
+
+## Cookies
+
+Where necessary, this website uses cookies to store information about a visitor’s preferences and history in order to better serve the user and/or present the user with customized content.
+
+## Advertisement and Other Third Parties
+
+Advertising partners and other third parties may use cookies, scripts and/or web beacons to track user activities on this website in order to display advertisements and other useful information. Such tracking is done directly by the third parties through their own servers and is subject to their own privacy policies. This website has no access or control over these cookies, scripts and/or web beacons that may be used by third parties. Learn how to [opt out of Google’s cookie usage](http://www.google.com/privacy_ads.html).
+
+## Links to Third Party Websites
+
+We have included links on this website for your use and reference. We are not responsible for the privacy policies on these websites. You should be aware that the privacy policies of these websites may differ from our own.
```

Comment:
Links may be shared that are outside compa through discussions, etc.

Can you include that?